internal/pkg/remote/manager.go

```diff
@@ -152,13 +152,6 @@ func (u *UnitOfWork) loadObject(object model.Object) (model.ObjectState, error)
 	// Set remote state
 	objectState.SetRemoteState(object)
 
-	// Generate local path
-	if !found {
```

Comment:
1. 
- v `remote.Manager`
- sa generuje `ObjectManifest` + path, ak neexistuje
- teda nacita sa nejaky `remote` object, a potrebujeme mu nastavit, aka cesta na disku mu zodpoveda
- ak uz existuje `local` state, tak existuje aj `manifest` a `path` -> takze to nie je potrebne

No a tu nastal problem, ze vsetky  `relations` su vytvorene, az ked dobehne `mapper/relations`, tj. po `OnObjectsLoad`. 

Na tomto mieste este nie su ziadne `relations`, takze sa path pre `variables` vygeneruje zle, ... kedze sa neurci spravne `parent` (`variables` su vnorene do konfiguracie, co ich pouziva).

Preto sa musi `path` generovat, az ked su `relations` nastavene, vid dalej.